src/TcxWriter.cpp

```diff
@@ -77,21 +77,8 @@ void TcxWriter::onWorkout(const WorkoutInfo &i)  {
     writer.startElement("Activity");
     writer. writeAttribute("Sport", sport);
     writer. writeElement("Id", i.start_time.format());
-    writer. startElement("Lap");
-    writer. writeAttribute("StartTime", i.start_time.format());
-    writer. writeElement("TotalTimeSeconds", "0");
-    writer. writeElement("DistanceMeters", "0");
-      {
-        std::ostringstream ss;
-        ss<< (int)(i.calories/100);
-        writer.writeElement("Calories", ss.str());
```

Comment:
The calories value stored was for the entire workout and hence it should
not be stored in the lap but should be stored in the activity.
Unfortunately, there doesn't really seem to be a suitable option in the
schema.  Should I put Calories as a note in the activity?

On Tue Feb 17 2015 at 01:25:08 mru <notifications@github.com> wrote:

> In src/TcxWriter.cpp
> <https://github.com/mru00/crane_gps_watch/pull/10#discussion_r24753096>:
>
> > @@ -77,21 +77,8 @@ void TcxWriter::onWorkout(const WorkoutInfo &i)  {
> >      writer.startElement("Activity");
> >      writer. writeAttribute("Sport", sport);
> >      writer. writeElement("Id", i.start_time.format());
> > -    writer. startElement("Lap");
> > -    writer. writeAttribute("StartTime", i.start_time.format());
> > -    writer. writeElement("TotalTimeSeconds", "0");
> > -    writer. writeElement("DistanceMeters", "0");
> > -      {
> > -        std::ostringstream ss;
> > -        ss<< (int)(i.calories/100);
> > -        writer.writeElement("Calories", ss.str());
>
> Where did the calories go?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mru00/crane_gps_watch/pull/10/files#r24753096>.
>